test/src/test/java/hudson/model/UpdateCenterMultiplePluginVersionsTest.java

```diff
@@ -0,0 +1,88 @@
+package hudson.model;
+
+/**
+ * Created by lvotypko on 12/20/17.
```

Comment:
Better to leave these out imo, `git blame` is more useful and already has the same info, although I do think you should add an MIT license header with your name here.